CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+# 13.0.0
+
+#### Breaking changes
+
+- In variables:
+  - Remove SimpleVariable and DatedVariable. Any Variable acts like old DatedVariable
```

Comment:
I don't think `AbstractVariable` was ever meant to be extended by child classes in country packages 🤔.
I don't think anyone ever did it anyway, so it's ok not to mention it.